test/OpenTelemetry.Tests/Implementation/Trace/TracerTest.cs

```diff
@@ -55,6 +56,183 @@ public void TracerStartReturnsNoOpSpanWhenNoSdk()
             Assert.False(span.IsRecording);
         }
 
+        [Fact]
+        public void Tracer_StartRootSpan_BadArgs_NullSpanName()
+        {
+            using var openTelemetry = Sdk.CreateTracerProvider(
```

Comment:
name is required parameter. so may be we are better off throw invalidargexception?

Reply:
Should we change the implementation?